source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -163,5 +154,45 @@ else if ((container = Container.dynamic_cast(baseCardElement)) == null)
         return containerView;
     }
 
+    public static void ApplyBleed(CollectionTypeElement collectionElement, LinearLayout collectionElementView, Context context, HostConfig hostConfig)
+    {
+        if (collectionElement.GetBleed() && collectionElement.GetCanBleed())
+        {
+            int padding = Util.dpToPixels(context, hostConfig.GetSpacing().getPaddingSpacing());
+            LinearLayout.LayoutParams layoutParams = (LinearLayout.LayoutParams) collectionElementView.getLayoutParams();
+            int marginLeft = layoutParams.leftMargin, marginRight = layoutParams.rightMargin;
+
+            ContainerBleedDirection bleedDirection = collectionElement.GetBleedDirection();
+            if (bleedDirection == ContainerBleedDirection.BleedToLeading || bleedDirection == ContainerBleedDirection.BleedToBothEdges)
+            {
+                marginLeft = -padding;
+            }
+
+            if (bleedDirection == ContainerBleedDirection.BleedToTrailing || bleedDirection == ContainerBleedDirection.BleedToBothEdges)
+            {
+                marginRight = -padding;
+            }
+
+            layoutParams.setMargins(marginLeft, layoutParams.topMargin, marginRight, layoutParams.bottomMargin);
+            collectionElementView.setLayoutParams(layoutParams);
+        }
+    }
+
+    public static void ApplyPadding(ContainerStyle elementContainerStyle, ContainerStyle parentContainerStyle, LinearLayout collectionElementView, Context context, HostConfig hostConfig)
+    {
+        if (elementContainerStyle != parentContainerStyle)
```

Comment:
just want to let you know that you can call GetPadding() on collectionType object to see if it gets padding.